packages/flutter/lib/src/material/data_table_theme.dart

```diff
@@ -223,6 +241,8 @@ class DataTableThemeData with Diagnosticable {
     properties.add(DoubleProperty('columnSpacing', columnSpacing, defaultValue: null));
     properties.add(DoubleProperty('dividerThickness', dividerThickness, defaultValue: null));
     properties.add(DoubleProperty('checkboxHorizontalMargin', checkboxHorizontalMargin, defaultValue: null));
+    properties.add(DiagnosticsProperty<MouseCursor>('headingCellCursor', headingCellCursor, defaultValue: null));
+    properties.add(DiagnosticsProperty<MaterialStateProperty<MouseCursor?>>('dataRowCursor', dataRowCursor, defaultValue: null));
```

Comment:
Isn't the type of this property `MaterialStateProperty<MouseCursor?>?` ?

Reply:
Good catch